plugins/gtav/gtav.cpp

```diff
@@ -101,22 +102,59 @@ void mumble_shutdownPositionalData() {
 bool mumble_fetchPositionalData(float *avatarPos, float *avatarDir, float *avatarAxis, float *cameraPos,
 								float *cameraDir, float *cameraAxis, const char **contextPtr,
 								const char **identityPtr) {
-	if (game->fetchPositionalData(avatarPos, avatarDir, avatarAxis, cameraPos, cameraDir, cameraAxis, contextPtr,
-								  identityPtr)
-		!= MUMBLE_PDEC_OK) {
+	for (uint8_t i = 0; i < 3; ++i) {
+		avatarPos[i] = cameraPos[i] = 0.f;
+		avatarDir[i] = cameraDir[i] = 0.f;
+		avatarAxis[i] = cameraAxis[i] = 0.f;
+	}
```

Comment:
Maybe a simple memset on these would make the intent clearer on first sight than using a loop in this way?

Reply:
[`std::fill_n`](https://en.cppreference.com/w/cpp/algorithm/fill_n)